packages/assets/scss/01-atoms/_figure.scss

```diff
@@ -24,6 +24,12 @@ $figures-gutter: 20px;
     margin: 10px $figures-gutter 10px $figures-gutter;
   }
 
+  @media ($bp-x-small-max) {
```

Comment:
Instead of adding a new media query, I wonder if there's any reasons of needing the previous one `@media ($bp-x-small-min)` to add the margins - the only thing I can think of is when the figure expands the full width on mobile and we want to remove the left-right margins.